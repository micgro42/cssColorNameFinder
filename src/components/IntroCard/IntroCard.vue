<template>
    <section class="introCard">
        <p>There are over a hundred named colors in CSS, but they are rarely used.</p>
        <p>
            This app helps you to use them more by showing names of colors that are similar to your favorite color. ðŸŽ¨
        </p>
    </section>
</template>

<script lang="ts">
import { defineComponent } from 'vue';

export default defineComponent({
    name: 'IntroCard',
});
</script>
