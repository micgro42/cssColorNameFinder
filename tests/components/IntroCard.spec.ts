import { shallowMount } from '@vue/test-utils';
import IntroCard from '../../src/components/IntroCard/IntroCard.vue';

describe( 'IntroCard', () => {
    it( 'matches snapshot', () => {
        const wrapper = shallowMount( IntroCard );
        expect( wrapper.element ).toMatchSnapshot();
    } );
} );
