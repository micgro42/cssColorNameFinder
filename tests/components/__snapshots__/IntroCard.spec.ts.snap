// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IntroCard matches snapshot 1`] = `
<section
  class="introCard"
>
  <p>
    
        There are over a hundred named colors in CSS, but they are rarely used.
    
  </p>
   
  <p>
    
        This app helps you to use them more by showing names of colors that are similar to your favorite color. ðŸŽ¨
    
  </p>
</section>
`;
